facebook/react-native
# Removed vulnerabilities from audit

## Summary


Description:
A `yarn audit` shows **17776 vulnerabilities found - Packages audited: 929983**

As mentioned in [https://github.com/facebook/react-native/issues/28082](url)
![image](https://user-images.githubusercontent.com/9755286/75400820-54f3ea00-58ce-11ea-8b80-d5b6f1b8e840.png)

React Native version: 0.61

## Changelog



[General] [Security] 

## Test Plan


1. run `yarn audit`
2. Observe  0 vulnerabilities
<img width="1076" alt="Screen Shot 2020-02-26 at 7 28 34 PM" src="https://user-images.githubusercontent.com/9755286/75400965-bae07180-58ce-11ea-9b2b-5c0cf70d5c53.png">
